src/main/java/com/oltpbenchmark/types/DatabaseType.java

```diff
@@ -47,19 +47,24 @@ public enum DatabaseType {
     SPANNER(false, true),
     SQLAZURE(true, true, true),
     SQLITE(true, false),
-    SQLSERVER(true, true, true),
+    SQLSERVER(true, true, true, true),
     TIMESTEN(true, false),
     PHOENIX(true, true);
 
 
-    DatabaseType(boolean escapeNames, boolean includeColNames, boolean loadNeedsUpdateColumnSequence) {
+    DatabaseType(boolean escapeNames, boolean includeColNames, boolean loadNeedsUpdateColumnSequence, boolean needsMonitoringPrefix) {
         this.escapeNames = escapeNames;
         this.includeColNames = includeColNames;
         this.loadNeedsUpdateColumnSequence = loadNeedsUpdateColumnSequence;
+        this.needsMonitoringPrefix = needsMonitoringPrefix;
+    }
+
+    DatabaseType(boolean escapeNames, boolean includeColNames, boolean loadNeedsUpdateColumnSequence) {
+        this(escapeNames, includeColNames, loadNeedsUpdateColumnSequence, false);
     }
 
     DatabaseType(boolean escapeNames, boolean includeColNames) {
-        this(escapeNames, includeColNames, false);
+        this(escapeNames, includeColNames, false, false);
```

Comment:
Any reason not to default that value to `true` everywhere for now?

Reply:
I think I'd prefer enabling that on a DBMS-by-DBMS basis, whenever the functionality has actually been implemented and tested. I think that makes more sense to me.

Reply:
Fair.  I tend to default to yes and let the CI catch errors and then disable it.  That way I at least get a warning that I need to implement something else when adding a new engine for instance.